coral-trino/src/test/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverterTest.java

```diff
@@ -235,7 +234,7 @@ public void testSupport(String trinoSql, String expectedRelString, String expect
     RelToTrinoConverter relToTrinoConverter = new RelToTrinoConverter();
     // Convert rel node back to Sql
     String expandedSql = relToTrinoConverter.convert(relNode);
-    assertEquals(expandedSql, expectedSql);
+    assertEquals(expectedSql, expandedSql);
```

Comment:
Could you also modify the `assertEquals` before it? (line 232)

Reply:
done. thanks for pointing out.